Source/Admin/Web.Angular/src/app/project/add-project/add-project.component.ts

```diff
@@ -14,23 +14,31 @@ import { AddProject, NationalSociety, User } from '../../shared/models';
 
 export class AddProjectComponent implements OnInit {
     name: string;
-    societies: Array<NationalSociety>;
-    owners: Array<User>;
+    societies: NationalSociety[];
+    owners: User[];
     selectedSociety: string;
     selectedOwner: string;
-    projectOwners: Array<User>;
+    projectOwners: User[];
```

Comment:
We could consider keeping data from the form fields within a `form` object, like this:

```js
formData: {
	selectedSociety: '',
	selectedOwner:  '',
	name: '',
	selectedSurveillanceOptionId: '',
}
```